packages/dapp-svelte-wallet/api/src/types.js

```diff
@@ -95,3 +95,24 @@
  * @property {any} requestContext
  * @property {string} id
  */
+
+/**
+ * @template T
+ * @typedef {Object} PetnameManager
+ * @property {(petname: Petname, object: T) => void} rename
```

Comment:
The client needs to know when the rename has taken effect, especially for unit tests.

```suggestion
 * @property {(petname: Petname, object: T) => Promise<void>} rename
```